packages/auto-id/src/keyManagement.ts

```diff
@@ -0,0 +1,290 @@
+import { KeyObject, createPrivateKey, createPublicKey, generateKeyPairSync } from 'crypto'
+import { promises as fs } from 'fs'
```

Comment:
What do you suggest as an alternative?

Reply:
It depend the usage that we have in mind for this package, if it's meant to be use by nodejs app (server setting, sure writting to file can be fine) but then the same package can't be use for client side, so we will likely need a second package for client side.

For client side, we won't write file for sure, just return the file in a text format (the pem) or store it to local storage

Reply:
> For client side, we won't write file for sure, just return the file in a text format (the pem) or store it to local storage

Actually, we would write.
The auto-kit's python code has the write to file functionality. So, I have kept it same as that of python's now.

Reply:
yes so this is what I'm saying, it depend on what is the purpose of this sdk, if it's use as part of a cli or in a node.js server setting, writing to fle is fine, but if we also plan for this same sdk and this same /auto-id to be use in a client setting, the use of fs won't fly as a browser won't allow the js code to directly write to the file system of the user.

If we compare it with the python-sdk the python code is run at runtime by the server and so you can write to file, but you are writing on a file in the server, not on the client side. But in the case of a website (react or next or other) the code will likely be run on client side (so you can write to local storage, but not to file, but you could create a file as buffer and offer it as download, but that won't be with fs) or if it's run on server it will likely be a serverless function so again no fs, you will instead create and drop the file in a bucket.

The issue is also that if a js file import fs in a context where fs is not allowed (client side browser) this will create runtime errors usually.

Reply:
Will make the necessary changes.